docs/detections/rules-ui-create.asciidoc

```diff
@@ -138,28 +137,31 @@ then:
 .. Use the filter and query fields to create the criteria used for detecting
 alerts.
 +
-NOTE: You can use {kib} saved queries (image:images/saved-query-menu.png[Saved query menu,17,17]) and queries from saved Timelines (*Import query from saved Timeline*) as rule conditions.
-+
-For example, the following rule detects when the `vssadmin delete shadows`
+The following example (based on the prebuilt rule <<prebuilt-rule-0-14-2-volume-shadow-copy-deleted-or-resized-via-vssadmin, Volume Shadow Copy Deleted or Resized via VssAdmin>>) detects when the `vssadmin delete shadows`
 Windows command is executed:
 
 ** *Index patterns*: `winlogbeat-*`
 +
-> Winlogbeat ships Windows event logs to {es-sec}.
+Winlogbeat ships Windows event logs to {elastic-sec}.
 
 ** *Custom query*: `event.action:"Process Create (rule: ProcessCreate)" and process.name:"vssadmin.exe" and process.args:("delete" and "shadows")`
 +
-> Searches the `winlogbeat-*` indices for `vssadmin.exe` executions with
+Searches the `winlogbeat-*` indices for `vssadmin.exe` executions with
 the `delete` and `shadow` arguments, which are used to delete a volume's shadow
 copies.
 +
 [role="screenshot"]
-image::images/rule-query-example.png[]
-+
-TIP: This example is based on the
-<<prebuilt-rule-0-14-2-volume-shadow-copy-deleted-or-resized-via-vssadmin, Volume Shadow Copy Deleted or Resized via VssAdmin>> prebuilt rule.
+image::images/rule-query-example.png[Rule query example]
+
+.. You can use {kib} saved queries (image:images/saved-query-menu.png[Saved query menu,18,18]) and queries from saved Timelines (*Import query from saved Timeline*) as rule conditions.
 +
-. Continue with <<preview-rules, previewing the rule>> (optional) or click **Continue** to <<rule-ui-basic-params, configure basic rule settings>>.
+When you use a saved query, the *Load saved query "_query name_" dynamically on each rule execution* check box appears:
+
+* Select this to use the saved query every time the rule runs. You won't be able to modify the rule's *Custom query* field or filters, because the rule will only use the saved query's settings. To make changes, modify the saved query itself.
+
+* Deselect this to load the saved query as a one-time way of populating the rule's *Custom query* field and filters. You can then make further adjustments to the rule's query and filters as needed. If the saved query is later changed, the rule will not inherit those changes.
```

Comment:
It might be useful to describe the first option as a "reference" or "link" to the saved query, and the second as a "copy" of the query and filters from the saved query.

Reply:
```suggestion
* Select this to use the saved query every time the rule runs. This links the rule to the saved query, and you won't be able to modify the rule's *Custom query* field or filters because the rule will only use settings from the saved query. To make changes, modify the saved query itself.

* Deselect this to load the saved query as a one-time way of populating the rule's *Custom query* field and filters. This merely copies the settings from the saved query to the rule, so you can then make further adjustments to the rule's query and filters as needed. If the saved query is later changed, the rule will not inherit those changes.
```